src/auslib/web/admin/views/rules.py

```diff
@@ -260,6 +260,24 @@ def _post(self, transaction, changed_by):
             if what.get("fallbackMapping") is not None and len(fallback_mapping_values) != 1:
                 return problem(400, "Bad Request", "Invalid fallbackMapping value. No release name found in DB")
 
+            alias = what.get("base_alias", None)
```

Comment:
I think the problem is that in _this_ part of the view this needs to be referred to as `alias` (not `base_alias`). I suggest you try printing out `what` and making a web request to confirm this.